build/document-extractor.js

```diff
@@ -279,77 +303,110 @@ function _addSingleSectionBCD($) {
+        }
       }
+      browserReleaseData.set(name, releaseData);
     }
-    browserReleaseData.set(name, releaseData);
-  }
 
-  for (const [key, compat] of Object.entries(data)) {
-    let block;
-    if (key === "__compat") {
-      block = compat;
-    } else if (compat.__compat) {
-      block = compat.__compat;
-    }
-    if (block) {
-      for (let [browser, info] of Object.entries(block.support)) {
-        // `info` here will be one of the following:
-        //  - a single simple_support_statement:
-        //    { version_added: 42 }
-        //  - an array of simple_support_statements:
-        //    [ { version_added: 42 }, { prefix: '-moz', version_added: 35 } ]
-        //
-        // Standardize the first version to an array of one, so we don't have
-        // to deal with two different forms below
-        if (!Array.isArray(info)) {
-          info = [info];
-        }
-        for (const infoEntry of info) {
-          const added = infoEntry.version_added;
-          if (browserReleaseData.has(browser)) {
-            if (browserReleaseData.get(browser).has(added)) {
-              infoEntry.release_date = browserReleaseData
-                .get(browser)
-                .get(added).release_date;
+    for (const [key, compat] of Object.entries(data)) {
+      let block;
+      if (key === "__compat") {
+        block = compat;
+      } else if (compat.__compat) {
+        block = compat.__compat;
+      }
+      if (block) {
+        for (let [browser, info] of Object.entries(block.support)) {
+          // `info` here will be one of the following:
+          //  - a single simple_support_statement:
+          //    { version_added: 42 }
+          //  - an array of simple_support_statements:
+          //    [ { version_added: 42 }, { prefix: '-moz', version_added: 35 } ]
+          //
+          // Standardize the first version to an array of one, so we don't have
+          // to deal with two different forms below
+          if (!Array.isArray(info)) {
+            info = [info];
+          }
+          for (const infoEntry of info) {
+            const added = infoEntry.version_added;
+            if (browserReleaseData.has(browser)) {
+              if (browserReleaseData.get(browser).has(added)) {
+                infoEntry.release_date = browserReleaseData
+                  .get(browser)
+                  .get(added).release_date;
+              }
             }
           }
         }
       }
     }
-  }
 
-  return [
-    {
-      type: "browser_compatibility",
-      value: {
-        title,
-        id,
-        isH3,
-        data,
-        query,
-        browsers,
+    return [
+      {
+        type: "browser_compatibility",
+        value: {
+          title,
+          id,
+          isH3,
+          data,
+          query,
+          browsers,
+        },
       },
-    },
-  ];
+    ];
+  } else if (specialSectionType === "specifications") {
+    let spec_urls = [];
```

Comment:
```suggestion
    let specURLs = [];
```
?

Reply:
I know it's called `spec_url` in BCD and now's not the time even try to change that. 

https://github.com/mdn/yari/blob/main/docs/conventions.md hopefully has something about the naming convention.